lib/templates/module/test/Gemfile.tt

```diff
@@ -1,9 +1,5 @@
 source "https://rubygems.org"
 
-<%=
-build_gemfile(
-  "terraspace",
-  "rspec-terraspace",
-  plugin_gem_name,
-)
-%>
+gem "terraspace", "~> <%= terraspace_minor_version %>"
```

Comment:
Yup. To avoid the circular dependency, terraspace is not in the gemspec. It's only in the Gemfile for development. https://github.com/boltops-tools/rspec-terraspace/blob/6deb6e7c3e48e94b785d52f2e34ad02ba3ec11bd/Gemfile#L10